ui/apps/platform/src/services/ClustersService.ts

```diff
@@ -102,6 +102,13 @@ export type AutoUpgradeConfig = {
     autoUpgradeFeature: 'SUPPORTED' | 'NOT_SUPPORTED';
 };
 
+/**
+ * Checks is auto upgrade is supported
+ */
+export function isAutoUpgradeSupported(autoUpgradeConfig: AutoUpgradeConfig) {
+    return autoUpgradeConfig.autoUpgradeFeature !== 'SUPPORTED';
+}
```

Comment:
This should be the opposite
```suggestion
export function isAutoUpgradeSupported(autoUpgradeConfig: AutoUpgradeConfig) {
    return autoUpgradeConfig.autoUpgradeFeature === 'SUPPORTED';
}
```